pandas/tests/tseries/offsets/conftest.py

```diff
@@ -23,3 +66,49 @@ def month_classes(request):
     Fixture for month based datetime offsets available for a time series.
     """
     return request.param
+
+
+@pytest.fixture(params=[getattr(offsets, o) for o in DATE_OFFSETS])
+def date_offset_types(request):
+    """
+    Fixture for all the datetime offsets available for a time series except business
+    subclasses.
+    """
+    return request.param
+
+
+@pytest.fixture(
+    params=[
+        getattr(offsets, o)
+        for o in DATE_OFFSETS
+        if issubclass(getattr(offsets, o), offsets.MonthOffset) and o != "MonthOffset"
+    ]
+)
+def date_month_classes(request):
```

Comment:
this seems to duplicate month_classes no? I like your name better, can you remove the other one (and change to use this one)